src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -369,6 +369,76 @@
     end
   end
 
+  describe 'POST #save_distribution' do
+    let(:action) { post :save_distributions, project: apache_project, distributions: ['openSUSE_Tumbleweed'] }
+
+    before do
+      login admin_user
+      Project.any_instance.stubs(:save_distributions).returns(true)
+      Project.any_instance.stubs(:prepend_kiwi_config).returns()
+      Project.any_instance.stubs(:store).returns(true)
+    end
+
+    it "should call 'add_distributions' with [ 'openSUSE_Tumbleweed' ]" do
+      Project.any_instance.expects(:save_distributions).with(['openSUSE_Tumbleweed']).returns(true).once
```

Comment:
What is testing this? that the already stubbed `save_distribuitions` returns true? Or that `action` is calling that stubbed method?